src/activeList.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable no-const-assign */
+const form = document.querySelector('#form');
+const inputF = document.querySelector('.inputF');
+const parentNode = document.querySelector('.parent-node');
+const enterErr = document.querySelector('.enter-err');
+const enterBtn = document.querySelector('.btn');
+
+let itemsArr = [];
+
+const tasksListShow = (itemsArr) => {
+  parentNode.innerHTML = '';
+  if (itemsArr.length > 0) {
+    itemsArr.forEach((item) => {
+      parentNode.insertAdjacentHTML('beforeend',
+        `<div class="show-list">
+      <input type="checkbox" class="check-box">
```

Comment:
- [ ] Based on project requirements for this milestone, Js file should have methods related to the status updates (`completed`: `true` / `false`). The status of the task should be updated in the ,local storage as well. Now even if I click on checkbox, the status remains the same `false`

https://user-images.githubusercontent.com/77038610/146576209-5325b894-4d68-43c5-ad81-f25c54a69fce.mp4

I suggest you to add  an event listener on the checkbox on `change` and update the status of the task in local storage 👍🏻